TOC-tidb-cloud.md

```diff
@@ -99,9 +99,12 @@
     - [Overview](/tidb-cloud/tidb-cloud-migration-overview.md)
     - [Configure Amazon S3 Access and GCS Access](/tidb-cloud/config-s3-and-gcs-access.md)
     - [Naming Conventions for Data Import](/tidb-cloud/naming-conventions-for-data-import.md)
-    - [Migrate from MySQL-Compatible Databases](/tidb-cloud/migrate-data-into-tidb.md)
-    - [Migrate Incremental Data from MySQL-Compatible Databases](/tidb-cloud/migrate-incremental-data-from-mysql.md)
-    - [Migrate and Merge MySQL Shards of Large Datasets](/tidb-cloud/migrate-sql-shards.md)
+    - [Migrate MySQL-Compatible Databases to TiDB Cloud Using Migration Jobs](/tidb-cloud/migration-job.md)
+    - [Migrate MySQL-Compatible Databases to TiDB Cloud](/tidb-cloud/migration-job.md)
```

Comment:
Should we delete this doc "Migrate MySQL-Compatible Databases to TiDB Cloud" when the migration job feature is released?

Reply:
By then maybe we can add a note in "Migrate MySQL-Compatible Databases to TiDB Cloud" to say that it is recommended to use Migration Jobs.

Reply:
OK updated.